src/contract-load.ts

```diff
@@ -18,7 +18,16 @@ export async function loadContract(arweave: Arweave, contractID: string) {
   const minFee = getTag(contractTX, 'Min-Fee');
   const contractSrcTX = await arweave.transactions.get(contractSrcTXID);
   const contractSrc = contractSrcTX.get('data', { decode: true, string: true });
-  const state = contractTX.get('data', { decode: true, string: true });
+
+  let state: string;
+  if (getTag(contractTX, 'initState')) {
+    state = getTag(contractTX, 'initState');
```

Comment:
Looking at the rest of the file, it seems `Init-State`, and `Init-State-TX` would fit better for names. Does that work with you @johnletey ?

Reply:
Ah yeah that'd match the SmartWeave tagging system better 😄

Will update this!